packages/sdk/src/swap/types.ts

```diff
@@ -83,8 +83,8 @@ interface SwapStatusResponseCommonFields extends ChainsAndAssets {
   estimatedDefaultDurationSeconds: number | undefined;
   srcChainRequiredBlockConfirmations: number | undefined;
   depositTransactionRef: string | undefined;
-  latestSwapScheduledAt: number | undefined;
-  latestSwapScheduledBlockIndex: string | undefined;
+  swapScheduledAt: number | undefined;
```

Comment:
the new designs show the time remaining for a swap, therefore we need to expose when it was first scheduled